src/containers/Admin/DashboardPickerTab/index.tsx

```diff
@@ -56,6 +56,15 @@ const DashboardPickerTab = (): JSX.Element => {
                     callback={(val): void => updateChoice(val)}
                     className="display-wrapper__display-card"
                 />
+                <RadioCard
+                    title="Holdeplass"
+                    description="Avgagnene viser kronologisk rekkefølge. Egner seg godt for holdeplasser og stopp. "
```

Comment:
Avgagnene -> Avgangene

Reply:
Ville gjerne fått fram i teksten også at den viser avvik og passar godt for _ett_ stopp. For eksempel:

> Her får avgangene full bredde med god plass til å vise avviksmeldinger. Egner seg godt for når man er interessert i ett eller få stopp.